src/node_api.cc

```diff
@@ -251,6 +252,18 @@ V8EscapableHandleScopeFromJsEscapableHandleScope(
   return reinterpret_cast<EscapableHandleScopeWrapper*>(s);
 }
 
+static
+napi_callback_scope JsCallbackScopeFromV8CallbackScope(
```

Comment:
The name is a bit of a misnomer, isn't it?

Reply:
It's consistent with the naming of the other similar functions.  We may want to change them all but I think that should be in a different PR.